demo/pages/alert.py

```diff
@@ -7,6 +7,19 @@
 
 dash.register_page(__name__)
 
+def callback_function(id):
```

Comment:
looks like the code is repeated many times, can we define this once and parameterize it to change whatever we need?

Reply:
so we don't have to re-define it on all sections